bc_obps/Makefile

```diff
@@ -0,0 +1,79 @@
+SHELL := /usr/bin/env bash
+PSQL=psql -h localhost
+DB_NAME=registration
+MANAGE_PY = manage.py
+
+help: ## Show this help.
+	@sed -ne '/@sed/!s/## //p' $(MAKEFILE_LIST)
+
+.PHONY: install_backend_asdf_tools
+install_backend_asdf_tools: ## install languages runtimes and tools specified in .tool-versions of the backend
+install_backend_asdf_tools:
+	@echo "Installing backend asdf tools"
+	@cat .tool-versions | cut -f 1 -d ' ' | xargs -n 1 asdf plugin-add || true
+	@asdf plugin-update --all
+	@#MAKELEVEL=0 is required because of https://www.postgresql.org/message-id/1118.1538056039%40sss.pgh.pa.us
+	@MAKELEVEL=0 POSTGRES_EXTRA_CONFIGURE_OPTIONS='--with-libxml' asdf install
+	@asdf reshim
+	@echo "Done installing backend asdf tools"
+
+.PHONY: install_poetry
+install_poetry: ## install poetry (MacOS/Linux only)
+install_poetry: ## poetry shell starts a new shell with the virtual environment activated
+	@echo "Installing poetry"
+	@curl -sSL https://install.python-poetry.org | python3 -
+	@poetry shell
+
+.PHONY: install_dev_tools
+install_dev_tools: ## install development tools
+install_dev_tools: stop_pg install_backend_asdf_tools install_poetry start_pg
+
+.PHONY: install_poetry_deps
+install_poetry_deps: ## install poetry dependencies
+install_poetry_deps:
+	@echo "Installing poetry dependencies"
+	@poetry install
+
+.PHONY: start_pg
+start_pg: ## start the database server if it is not running
+start_pg:
+	@pg_ctl status || pg_ctl start
+
+.PHONY: stop_pg
+stop_pg: ## stop the database server. Always exits with 0
+stop_pg:
+	@pg_ctl stop; true
+
+.PHONY: create_db
```

Comment:
not really sure how makefiles work, but I think this one doesn't start the server? If I run it without having the server started, I get an error. If server-starting shouldn't be part of the makefile, then I think it should be added as a step to the readme before running this `make`